katalog/cert-manager/README.md

```diff
@@ -1,34 +1,72 @@
 # cert-manager
+
 cert-manager is an automation tool for management and issuence of TLS
 certificates from various issuing resource in a Kubernetes native way. It
 ensures that certificates are valid and attempts to renew them before expiry.
 
 This package deploys cert-manager to be used with [Let's
 Encrypt](https://letsencrypt.org/) Certificate Authority.
 
-
 ## Requirements
-- Kubernetes >= `1.10.0`
-- Kustomize >= `v1`
-- [nginx-ingress]()
 
+-   Kubernetes >= `1.10.0`
+-   Kustomize >= `v1`
+-   [nginx-ingress]()
 
 ## Image repository and tag
-* Cert Manager image: `quay.io/jetstack/cert-manager-controller:v0.9.0`
-* Cert Manager repo: https://github.com/jetstack/cert-manager
-* Cert Manager documentation: https://docs.cert-manager.io/en/release-0.9/index.html
 
+-   Cert Manager image: `quay.io/jetstack/cert-manager-controller:v0.9.0`
+-   Cert Manager repo: https://github.com/jetstack/cert-manager
+-   Cert Manager documentation: https://docs.cert-manager.io/en/release-0.9/index.html
 
 ## Configuration
+
 Fury distribution cert-manager is deployed with following configuration:
-- Default issuer kind is ClusterIssuer
-- Default issuer is letsencrypt
+
+-   Default issuer kind is ClusterIssuer
+-   Default issuer is letsencrypt
 
 ## Deployment
-You can deploy Cert Manager by running following command in the root of the project:
+
+The deployment is the following, but keep in mind that, depending of `nginx` or `dual-nginx` you need to specify the class name accordingly.
+To do that you need to patch the ClusterIssuer resource in order to keep it synced with your specific deployment.
+So before proceed to the build and apply, you should provide a patchesJson6902 like the following:
+
+```yml
+patchesJson6902:
+  - target:
+      group: certmanager.k8s.io
+      version: v1alpha1
+      kind: ClusterIssuer
+      name: letsencrypt-staging
+    path: patches/dual-nginx.yml
+  - target:
+      group: certmanager.k8s.io
+      version: v1alpha1
+      kind: ClusterIssuer
+      name: letsencrypt-prod
+    path: patches/dual-nginx.yml
+```
+
+and under the `patches/dual-nginx.yml`:
+
+```yml
+---
+- op: "replace"
+  path: "/spec/acme/solvers/0/http01/ingress/class"
+  value: "dual-nginx"
```

Comment:
This value should match the ingress class, which is set to `external` in the dual-nginx module: https://github.com/sighupio/fury-kubernetes-ingress/blob/66d2bda798a1026652dfe0dd951c05f34245876d/katalog/dual-nginx/external/patch/daemonset.yml#L11

Reply:
you're totally right @lnovara , was my fault. I fix i right now!